.github/workflows/deploy.py

```diff
@@ -79,15 +79,21 @@ def check_neon_evm_tag(tag):
             f"evm_loader image with {tag} tag isn't found. Response: {response.json()}")
 
 
-def update_neon_evm_tag_if_same_branch_exists(branch, neon_evm_tag):
-    if branch != "":
+def is_neon_evm_branch_exist(branch):
+    if branch != "" and branch is not None:
```

Comment:
simplify to:
```python
if branch:
    ...
```
because
```python
In [1]: branch = ""

In [2]: if branch:
   ...:     print("branch exist")
   ...:

In [3]: branch = None

In [4]: if branch:
   ...:     print("branch exist")
   ...:

In [5]: branch = "dsfsd"

In [6]: if branch:
   ...:     print("branch exist")
   ...:
branch exist
```